sdcclient/_monitor.py

```diff
@@ -306,6 +306,46 @@ def get_dashboards(self):
         res = requests.get(self.url + self._dashboards_api_endpoint, headers=self.hdrs, verify=self.ssl_verify)
         return self._request_result(res)
 
+    def get_dashboard(self, dashboard_id):
+        '''**Description**
+            Return a dashboard with the pased in ID. This includes the dashboards created by the user and the ones shared with them by other users.
+
+        **Success Return Value**
+            A dictionary containing the requested dashboard data.
+
+        **Example**
+            `examples/dashboard_basic_crud.py <https://github.com/draios/python-sdc-client/blob/master/examples/dashboard_basic_crud.py>`_
+        '''
+        id_str = dashboard_id if isinstance(dashboard_id, type(str)) else str(dashboard_id)
```

Comment:
I'd recommend `isinstance(dashboard_id, basestring)` instead, mostly for consistency with few other occurrences of the same check in the library.

See https://stackoverflow.com/a/4843178/3426592 for Python 2.x.

**Note**: It will (magically) work on Python 3.x thanks to this trick: https://github.com/draios/python-sdc-client/blob/28a1e913a7cde872b47e2b45925de7d3f7e0a807/sdcclient/_monitor.py#L11

---

However, instead of assigning the extra variable I'd rather suggest constructing the URL for the following line as:
```
self.url + self._dashboards_api_endpoint + "/" + str(dashboard_id)
```

since `str(42) == str('42')`. It may add a little overhead if you pass a string already, but IMHO it looks a little more readable and remains consistent with what's done elsewhere in the library.